src/hearings/store/reducers/hearing-request.reducer.ts

```diff
@@ -45,20 +48,21 @@ export const initialHearingRequestState: HearingRequestStateData = {
   lastError: null,
 };
 
-export function hearingRequestReducer(currentState = initialHearingRequestState,
-                                      action: fromActions.HearingRequestAction): HearingRequestStateData {
+export function hearingRequestReducer(
+  currentState = initialHearingRequestState,
+  action: fromActions.HearingRequestAction): HearingRequestStateData {
   switch (action.type) {
     case fromActions.RESET_HEARING_REQUEST: {
       return {
         ...initialHearingRequestState
       };
     }
-    case fromActions.UPDATE_HEARING_REQUEST: {
+    case fromActions.NAVIGATE_BEGINNING_CANCEL_REQUEST:
+    case fromActions.UPDATE_HEARING_REQUEST:
```

Comment:
this hearing request reducer is used for request hearing only, the cancel shouldn't use ngrx store.